cssom-view-1/Overview.bs

```diff
@@ -1070,21 +1070,16 @@ result of running these steps:
         <dt><a>caret range</a>
         <dd>null
     </dl>
-1. Otherwise, return a <a>caret position</a> where the
-    <a>caret range</a> is a collapsed
-    {{Range}} object for the position
-    where the text insertion point indicator would have been inserted when applying the <a>transforms</a> that apply to the descendants of the
-    <a>viewport</a>, and the other properties are set as follows:
-
-    <dl>
-        <dt><a>caret node</a>
-        <dd>The [=range/start node=]
-            of the <a>caret range</a>.
-
-        <dt><a>caret offset</a>
-        <dd>The [=range/start offset=] of
-            the <a>caret range</a>.
-    </dl>
+1. Otherwise:
+
+    1. Let <var>caretRange</var> to be a collapsed {{Range}} object for the position where the text insertion point indicator would have been inserted when applying
```

Comment:
The [caret range](https://drafts.csswg.org/cssom-view/#caret-range) concept doesn't seem particularly well defined in the spec. What's the idea and is it really necessary?

Reply:
The [caret range](https://drafts.csswg.org/cssom-view/#caret-range) is used in [`CaretPosition::getClientRect`](https://www.w3.org/TR/cssom-view/#dom-caretposition-getclientrect.). Implementation details can be to create the live range on demand when the `getClientRect` method is invoked.